src/app/utils/item.js

```diff
@@ -138,8 +134,9 @@ function LibraryItem() {
       suppressed,
       barcode,
       itemSource: mappedItemSource,
-      isRecap: isRecap || isOffsite,
```

Comment:
We need `isRecap` but `isOffsite` should not be part of that check. I added it as a different field in case we need it but it looks like we might not.